srml/balances/src/tests.rs

```diff
@@ -84,13 +83,13 @@ fn lock_removal_should_work() {
 		.build()
 		.execute_with(|| {
 			Balances::set_lock(
-				ID_1, 
-				&1, 
+				ID_1,
+				&1,
 				WithdrawLock::Normal(NormalLock {
-					amount: u64::max_value(), 
+					amount: u64::max_value(),
 					until: u64::max_value(),
-				}), 
-				WithdrawReasons::all()
+				}),
+				WithdrawReasons::all(),
```

Comment:
Are these code format rules change?

Reply:
These file had been modified by @readlnh in #141.

I think we should make a git-hook to keep `cargo fmt`.